buildSrc/src/main/kotlin/jvm-module.gradle.kts

```diff
@@ -148,6 +148,9 @@ fun Module.addDependencies() = dependencies {
     compileOnlyApi(JavaX.annotations)
     ErrorProne.annotations.forEach { compileOnlyApi(it) }
 
+    implementation(Spine.Logging.lib)
+    runtimeOnly(Spine.Logging.backend)
```

Comment:
This is a problematic place. All Spine `jvm-module`s should have `spine-logging`, but not `spine-logging` itself.